tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -918,6 +926,7 @@ func init() {
     actionInvokeCmd.Flags().BoolVarP(&flags.action.result, "result", "r", false, wski18n.T("blocking invoke; show only activation result (unless there is a failure)"))
 
     actionGetCmd.Flags().BoolVarP(&flags.common.summary, "summary", "s", false, wski18n.T("summarize action details"))
+    actionGetCmd.Flags().BoolVarP(&flags.action.url, "url", "r", false, wski18n.T("get action url"))
```

Comment:
Updated the command usage to inform the user that `--url`, `--summary`, and field filtering do not work in conjunction with each other. I'm think we need to go through all the commands and see which flags work together and throw errors when multiple flags that incompatible with each other are specified.